packages/chaire-lib-backend/src/utils/processManagers/OSRMServicePreparation.ts

```diff
@@ -12,12 +12,14 @@ import { fileManager } from '../filesystem/fileManager';
 import { getOsrmDirectoryPathForMode, defaultDirectoryPrefix, getDirectoryPrefix } from './OSRMServicePath';
 import { RoutingMode } from 'chaire-lib-common/lib/config/routingModes';
 
-const defaultImportOsmFilePath = fileManager.directoryManager.getAbsolutePath('imports/network.osm');
```

Comment:
Is it moved somewhere else?

Reply:
No, it's gone, that was not even a real default (maybe once, but not anymore, it's not the name the tasks give it).